pkg/virt-operator/kubevirt_test.go

```diff
@@ -1404,13 +1524,17 @@ var _ = Describe("KubeVirt Operator", func() {
 			Expect(kv.Status.Phase).To(Equal(v1.KubeVirtPhaseDeploying))
 			Expect(len(kv.Status.Conditions)).To(Equal(0))
 
-			// 2 because waiting on controller and virt-handler daemonset until API server deploys successfully
-			expectedUncreatedResources := 2
+			// 3, 2 because waiting on controller and virt-handler daemonset until API server deploys successfully
```

Comment:
Maybe it's just me, but I had to read this 3 times before understanding that this means 3 = 2 + 1 :laughing:

Reply:
Yeah, right...

Reply:
Better?
```
// 3 in total are yet missing at this point
// because waiting on controller, controller's PDB and virt-handler daemonset until API server deploys successfully
```

Reply:
yes :)